### DISCUSSION TITLE
[Auth0 client] error host

### DISCUSSION THREAD

**Discussion by georgev-97** [mia-platform/community#382] [Q&A] [answered]
### Which section?

Console

### Which is the version?

12

### Description

Hello guys,

I keep getting this error on my auth0-client:

```
{"error":"host not found","level":50,"msg":"error getting original host","reqId":"3090ac91-7210-4680-988c-cf9bf78c0aaf","time":1706692245461}
{"host":{"hostname":"auth0-client"},"http":{"request":{"method":"POST","userAgent":{"original":"axios/1.6.0"}},"response":{"statusCode":400,"body":{"bytes":0}}},"level":30,"msg":"request completed","reqId":"3090ac91-7210-4680-988c-cf9bf78c0aaf","responseTime":0,"time":1706692245461,"url":{"path":"/oauth/token"}}
```

I have a custom microservice on mia-platform that is exposed publicly, it gets the requests from a frontend and calls internally the correct auth0-client APIs.
I don't have this error if I call the auth0-client directly with postman.

**Comment by malta895** [comment, marked as answer]
Hi @georgev-97,
is the service forwarding a `X-Forwarded-Host` header to the Auth0-client?

**Reply by georgev-97** [reply]
not that I know of, I have an envoy-api-gateway, maybe it is adding the header by default

**Reply by malta895** [reply]
Yes it is probably adding the header when calling your service, but your service in turn has to forward it when calling the `auth0-client`. Can you make sure it does?

**Reply by fredmaggiowski** [reply]
Hi, Yes I can confirm that Envoy API Gateway will forward it from your Ingress controller to your microservice. As @malta895 said, your microservice then needs to propagate it to the auth0-client.

**Reply by fredmaggiowski** [reply]
I've looked through the documentation and did not found any detail about this; I think this use case could be easily documented @malta895 will you please track the activity in our internal backlog?

@marcofilippi FYI

**Reply by georgev-97** [reply]
Thanks, I will let you know if this works!

**Reply by georgev-97** [reply]
It works fine!
